zombienet/tests/manual/001_runtime_upgrade.README.md

```diff
@@ -0,0 +1,141 @@
+# Check if runtime upgrade works:
+
+## 1. Build polkadot/kusama with sudo and fast-runtime:
+```
+# checkout https://github.com/paritytech/polkadot/tree/it/release-v0.9.38-fast-sudo
+cargo build --release --features fast-runtime
+cp ./target/release/polkadot ~/local_runtime_testing/bin/
+```
+
+## 2. Prepare runtimes to start with:
+
+```
+cd <cumulus-git-repo-dir>
+# skip if, you already have binary
+# cargo build --release -p polkadot-parachain-bin
+```
+
+or
+
+```
+Check docker image, e.g.: `paritypr/polkadot-parachain-debug:release-parachains-v9380-2a051cc6`
+https://hub.docker.com/r/paritypr/polkadot-parachain-debug/tags
+```
+
+## 3. Prepare new runtimes for upgrade:
+
+Either build your own wasm locally or use `srtool`:
+
+Run manual build for some branch, e.g.: `draft-polkadot-v0.9.39` and `statemine-local` and download artifacts
+https://github.com/paritytech/srtool/actions/workflows/manual.yml
+
+![](./srtool-manual.png)
+
+
+## 4. Do test:
+
+e.g: kusama-local + statemine-local
+
+### 1. Starting parachains from local binary
+```
+cd <cumulus-git-repo-dir>
+
+POLKADOT_BINARY_PATH=~/local_runtime_testing/bin/polkadot \
+POLKADOT_PARACHAIN_IMAGE= \
+POLKADOT_PARACHAIN_CMD=./target/release/polkadot-parachain \
+RELAY_CHAIN=kusama-local \
+PARA_CHAIN=statemine-local \
+	~/local_runtime_testing/bin/zombienet-linux --provider native spawn ./zombienet/tests/manual/parachain-for-upgrade.toml
+```
+
+alternative: starting parachains from docker `paritypr/polkadot-parachain-debug:release-parachains-v9380-2a051cc6`
+(TODO: does not work, need to check command or whatever)
+```
+cd <cumulus-git-repo-dir>
+
+POLKADOT_BINARY_PATH=~/local_runtime_testing/bin/polkadot \
+POLKADOT_PARACHAIN_IMAGE=paritypr/polkadot-parachain-debug:release-parachains-v9380-2a051cc6 \
+POLKADOT_PARACHAIN_CMD= \
+RELAY_CHAIN=kusama-local \
+PARA_CHAIN=statemine-local \
+	~/local_runtime_testing/bin/zombienet-linux --provider native spawn ./zombienet/tests/manual/parachain-for-upgrade.toml
+```
+
+### 2. Wait for parachain to start producing blocks:
+
+### 3. Do some operations:
```

Comment:
any reason the below steps are not implemented as an automatic test script?

Reply:
well, this is supposed to be kind of example, and these operations should vary,
for example on BridgeHub runtimes we wont have pallet_assets/pallet_uniques

and also all steps/everything is able to be scripted: https://github.com/paritytech/cumulus/issues/2264

anyway, thank you for any feedback :)